lib/dbutils.js

```diff
@@ -431,7 +425,50 @@ dbu.buildGetQuery = function(tableName, query, schema, includePreparedForDelete)
         }
         sql += condition + constructOrder(query, schema) + limit;
     }
-    return {sql: sql, params: params};
+    return sql;
+};
+
+function extractConditionParams(query, schema) {
```

Comment:
Not sure if this makes sense, but possibly useful to keep in mind for future work / Cassandra: Both sqlite and cassandra also support named parameters. When using those, we could pass in the attributes object mostly as-is (modulo complex predicates, which would require some naming scheme).